cmd/pod-scaler/main.go

```diff
@@ -212,9 +214,20 @@ func mainProduce(opts *options, cache cache) {
 }
 
 func mainAdmission(opts *options) {
-	restConfig, err := rest.InClusterConfig()
+	// TODO: figure out the logrusr dependency hell with logr and c-r
```

Comment:
@alvaroaleman do you have any suggestions here? Importing `logrusr` (a shim to allow `logrus` to be the `logr` logger used for `controller-runtime`) pulls in an incompatible version of `logr` and causes the `controller-runtime` vendors to not compile :|

Reply:
Copy the shim or `replace` the `logr` version